README.md

```diff
@@ -10,6 +10,23 @@ devenv shell
 
```

Comment:
Now that we have gitsubmodules, make sure to also comment the proper cloning here.

Also, maybe a quick explanation of why npm is needed here makes sense.